README.md

```diff
@@ -4,105 +4,215 @@ Universum
 [![Build Status](https://travis-ci.org/serokell/universum.svg?branch=master)](https://travis-ci.org/serokell/universum)
 [![Hackage](https://img.shields.io/hackage/v/universum.svg)](https://hackage.haskell.org/package/universum)
 
-Prelude used in Serokell.
+Custom _Prelude_ used in Serokell.
```

Comment:
I think `Prelude` refers to a particular module, not to the concept, so writing “Custom prelude used in Serokell” would've looked more natural (at least for me).